### PR TITLE
[chef][windows] allow custom prefix when specifying agent artifact+ve…

### PR DESCRIPTION
**Pull request by truthbk**
…rsion

### CODE REVIEW

File: spec/dd-agent_spec.rb [hunk 1]
```diff
@@ -268,11 +268,12 @@ def set_env_var(name, value)
           :file_cache_path => 'C:/chef/cache'
         ) do |node|
           node.normal['datadog'] = {
-            'agent6' => false,
+            'agent6' => true,
             'api_key' => 'somethingnotnil',
             'agent6_version' => {
               'debian' => '1:6.9.0-1',
               'rhel' => '6.9.0-1',
+              'windows' => '6.9.0'
```
Comment (olivielpeau) [inline spec/dd-agent_spec.rb]: nit, you can ignore: trailing commas are nice :)

 ```suggestion
               'windows' => '6.9.0',
```

File: metadata.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures datadog components'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '3.1.0'
+version          '3.1.1'
```
Comment (olivielpeau) [inline metadata.rb]: version bump should be removed from PR

File: recipes/_install-windows.rb [hunk 3]
```diff
@@ -40,8 +40,11 @@
   dd_agent_latest = 'ddagent-cli-latest'
 end
 
+# If a different file prefix is specified, use that:
+dd_agent_installer_prefix = node['datadog']['windows_installer_prefix'] ? node['datadog']['windows_installer_prefix'] : 'ddagent-cli'
```
Comment (olivielpeau) [inline recipes/_install-windows.rb]: The attribute `windows_installer_prefix` must be added to `attributes/default.rb`, and documented there.

Also, nit: for consistency with other Windows-related attributes, I slightly prefer a name such as `windows_agent_installer_prefix`, even though it's longer

File: metadata.rb (hunk 2 above)
Comment (truthbk) [reply metadata.rb]: Yeah, we're using this from the QA env, so I needed the bump. But we'll remove that before merging.

Comment (remeh) [reply metadata.rb]: Let's remove this and merge to `master`!

**Pull request merged**